pkgs/development/libraries/libgourou/default.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchzip, pugixml, updfparser, curl, openssl, libzip  }:
+
+stdenv.mkDerivation rec {
+  name = "libgourou";
+  version = "0.8.1";
+
+  src = fetchzip {
+    url = "https://indefero.soutade.fr/p/${name}/source/download/v${version}/";
+    sha256 = "sha256-X56K4z1+k62Q9pxnN8kx35oc7+uJJN/o/tpnKx1FjwE=";
+    extension = "zip";
+  };
+
+  postPatch = ''
+    patchShebangs scripts/setup.sh
+  '';
+
+  configurePhase = ''
+    mkdir lib
+    ln -s ${updfparser}/lib lib/updfparser
+  '';
+
+  buildInputs = [ pugixml updfparser curl openssl libzip ];
+
+  buildPhase = ''
+    make BUILD_STATIC=1 BUILD_SHARED=1
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dt $out/include include/libgourou*.h
+    install -Dt $out/lib libgourou.so
+    install -Dt $out/lib libgourou.so.${version}
+    install -Dt $out/lib libgourou.a
+    install -Dt $out/bin utils/acsmdownloader
+    install -Dt $out/bin utils/adept_{activate,loan_mgt,remove}
+    install -Dt $out/share/man/man1 utils/man/*.1
```

```suggestion
    installManPage utils/man/*.1
```

Needs also `installShellFiles` in `nativeBuildInputs`.